src/rules/rule-non-nested-empty-line-before/__tests__/index.js

```diff
@@ -49,6 +49,28 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
+  config: [ "always", { except: ["after-single-line-comment"] } ],
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: "/**\n * comment\n*/\n\na {}",
+  }, {
+    code: "/**\n\t* comment\n*/\na {}",
```

Comment:
Why does this one pass? Seems like it should fail since it's a multiline comment?